OpenTabletDriver.Configurations/Configurations/XP-Pen/Artist 24.json

```
@@ -0,0 +1,32 @@
+{
+  "Name": "XP-Pen Artist 24",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 526.85,
+      "Height": 296.35,
+      "MaxX": 105370,
+      "MaxY": 59270
+    },
+    "Pen": {
+      "MaxPressure": 8192,
```

Double check this in the debugger please.
```suggestion
      "MaxPressure": 8191,
```

---

@Tenchrio I see that you've done the rest but you haven't looked at this yet, could you confirm that its actually 8191?

---

Yes, it does, I think I had it as the value since yesterday but changed it last night by mistake, changed it back to 8191 locally and this is what it said in the debugger when pressing down hard:
`{ 02 A1 8B 1F E0 A2 FF 1F 20 11 00 00 }, Delta:0.074ms, Position:[8075,41696], Pressure:8191, PenButtons:[False False], Eraser:False, Tilt:[32,17], ReportType:XP_PenTabletOverflowReport`